literature.bib

```diff
@@ -13948,6 +14679,46 @@ @article{BSR04
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2003 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@techreport{C03,
+	title = {{Predicting When Product Line Investment Pays}},
+	author = {Cohen, Sholom},
+	institution = {Carnegie-Mellon University},
+	address = {PY, USA},
+	year = 2003,
+}
+
+@incollection{SS:BOICCAD03
+	title = {{GRASP - A New Search Algorithm for Satisfiability}},
+	author = {Silva, Jo{\~a}o P. Marques and Sakallah, Karem A.},
+	booktitle = {The Best of ICCAD},
+	pages = {73--89},
+	publisher = Springer,
+	address = Denver,
+	year = 2003,
```

Comment:
rc

Reply:
BTW you'll face the same issue with MyShort + Booktitles...